### ISSUE TITLE
Error "Cannot read property 'name' of undefined"

### ISSUE THREAD

**Issue by aiwilliams** [JupiterOne/graph-duo#6]
```
err.message
Cannot read property 'name' of undefined
err.name
TypeError
err.stack
TypeError: Cannot read property 'name' of undefined
at Object.convertAccount (/var/task/dist/node_modules/@jupiterone/graph-duo/dist/converter/index.js:10:53)
at Object.executionHandler (/var/task/dist/node_modules/@jupiterone/graph-duo/dist/steps/fetch-users/index.js:16:43)
at processTicksAndRejections (internal/process/task_queues.js:97:5)
at async executeStep (/var/task/dist/node_modules/@jupiterone/integration-sdk/src/framework/execution/dependencyGraph.js:160:17)
at async run (/var/task/dist/node_modules/p-queue/dist/index.js:255:29)
level
50
msg
Step failed to complete due to error.
```

**Comment by aiwilliams** [comment]
Looks like the converter is being handing an `undefined` `data`:
```
      assign: {
        ...convertProperties(data),
        _key: `duo-account:${siteId}:${data.name
          .toLowerCase()
          .replace(/\s/, '-')}`,
        _type: 'duo_account',
        _class: 'Account',
        name: data.name,
        siteId,
        displayName: `${data.name} Duo Account`,
        webLink: `https://admin-${siteId}.duosecurity.com/`,
      },
```

**Comment by aiwilliams** [comment]
This has been resolved by #8. Once the validation ensures we have access to settings, there should not be an `undefined` JSON body later on during ingestion.
